i8_terminal/commands/screen/screen_losers.py

```diff
@@ -0,0 +1,85 @@
+from typing import Any, Dict, Optional
+
+import click
+import investor8_sdk
+import pandas as pd
+from rich.console import Console
+
+from i8_terminal.commands.screen import screen
+from i8_terminal.common.cli import pass_command
+from i8_terminal.common.layout import df2Table
+from i8_terminal.common.metrics import (
+    get_current_metrics_df,
+    prepare_current_metrics_formatted_df,
+)
+from i8_terminal.common.utils import export_data, export_to_html
+from i8_terminal.config import APP_SETTINGS
+from i8_terminal.types.market_indice_param_type import MarketIndiceParamType
+from i8_terminal.types.metric_view_param_type import MetricViewParamType
+
+
+def prepare_losers_df(index: str, view_name: Optional[str]) -> Optional[pd.DataFrame]:
+    metrics = APP_SETTINGS["commands"]["screen_losers"]["metrics"]
+    companies_data = investor8_sdk.ScreenerApi().get_top_stocks("losers", index=index)
+    if companies_data is None:
+        return None
+    companies = [company.ticker for company in companies_data]
+    if view_name:
+        metrics = metrics + "," + APP_SETTINGS["metric_view"][view_name]["metrics"]
+    return get_current_metrics_df(",".join(companies), metrics)
+
+
+@screen.command()
+@click.option(
+    "--index",
+    "-i",
+    type=MarketIndiceParamType(),
+    default="$SPX",
+    help="Index of market.",
+)
+@click.option(
+    "--view_name", "view_name", "-v", type=MetricViewParamType(), help="Metric view name in configuration file."
+)
+@click.option("--export", "export_path", "-e", help="Filename to export the output to.")
+@pass_command
+def losers(index: str, view_name: Optional[str], export_path: Optional[str]) -> None:
+    """
+    Lists today loser companies.
+
+    Examples:
+
+    `i8 screen losers --index $SPX`
+
+    """
+    console = Console()
+    with console.status("Fetching data...", spinner="material"):
+        df = prepare_losers_df(index, view_name)
+    if df is None:
+        console.print("No data found for today losers", style="yellow")
+        return
+    columns_justify: Dict[str, Any] = {}
+    if export_path:
+        if export_path.split(".")[-1] == "html":
```

Comment:
This whole block (from line 61 to line 85) is exactly the same content as the other file (duplicate code!). Please move them to a function in the common layer.